community/code_of_conduct.md

```diff
@@ -1,43 +1,80 @@
 
-In the interest of fostering an open and welcoming environment, we pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, nationality, personal appearance, race, religion, or sexual identity and orientation.
+We strive to create a welcoming and inclusive culture that empowers people to provide outstanding public service. That kind of atmosphere requires an open exchange of ideas balanced by thoughtful guidelines. If we have only openness, colleagues who are on the receiving end of thoughtless or intentionally hurtful comments and behavior may reasonably withdraw. If we have overly intrusive guidelines, people may feel unwelcome.
 
-## Our Standards
+## Our standards
 
-Examples of behavior that contributes to creating a positive environment include:
+It would be impossible to list everything staff can do to create a more welcoming space, and we know this team will find ways to include their colleagues that we haven't even thought of. But when in doubt, we encourage you to look to these principles for guidance:
 
-- Using welcoming and inclusive language
-- Being respectful of differing viewpoints and experiences
-- Gracefully accepting constructive criticism
-- Focusing on what is best for the community
-- Showing empathy towards other community members
+* Practice empathy and humility.
+* Assume competence first.
+* Assume that everyone we work with is doing their best work for the community.
+* Listen carefully and actively.
+* Ask questions, and seek to understand your partners' context.
+* Encourage other people to listen as much as they speak.
+* Treat other people's identities and cultures with respect. Make an effort to say people's names correctly and refer to them by their chosen pronouns.
+* Using welcoming and inclusive language
+* Being respectful of differing viewpoints and experiences
+* Gracefully accepting constructive criticism
+* Focusing on what is best for the community.
+* Showing empathy towards other community members.
 
-Examples of unacceptable behavior by participants include:
+## Unacceptable behavior
 
-- The use of sexualized language or imagery and unwelcome sexual attention or advances
-- Trolling, insulting/derogatory comments, and personal or political attacks
-- Public or private harassment
-- Publishing other's private information, such as a physical or electronic address, without explicit permission
-- Other conduct which could reasonably be considered inappropriate in a professional setting
+To help colleagues understand the kinds of behaviors that are illegal or run counter to the culture we seek to foster, we've listed below actions that are counter to our values.  We've also included steps to take if you encounter behavior that runs contrary to this policy.
```

Comment:
I think a word like prohibited is a better choice than "illegal", none of us are Judge Dredd.

Reply:
Yup, removed